lib/ignore/ignore.go

```diff
@@ -125,40 +127,42 @@ func New(fs fs.Filesystem, opts ...Option) *Matcher {
 }
 
 func (m *Matcher) Load(file string) error {
+	if m == nil {
+		return IsNil
+	}
```

Comment:
well I'd panic anyway if we just removed a check with a nil pointer dereference.